index.src.html

```diff
@@ -495,24 +511,28 @@ <h4 id="generate-report" export algorithm>Generate report of |type| with
   |data|</h4>
 
   When the user agent is to <dfn>generate a report</dfn> for a <a>Document</a>
-  (|document|), given a string (|type|), another string (|destination|), and
-  a serializable object (|data|), it must run the following steps:
+  or {{WorkerGlobalScope}} object (|context|), given a string (|type|), another
+  string (|destination|), and a serializable object (|data|), it must run the
+  following steps:
+
+  1.  If |context| is a <a>Document</a>, then let |settings| be |context|'s
+      [=environment settings object=].
```

Comment:
relevant settings object should work for both. This link just goes to the general definition of environment settings object, which isn't right.

Reply:
Thanks, updated